docs/schedule/best-practices.ko.md

```diff
@@ -73,15 +73,16 @@ __예:__
+|agency_id | 피드에 대행사가 하나만 있는 경우에도 포함되어야 합니다. (포함할 권장 사항도 참조하십시오.`agency_id` \~에[agency.txt](#agencytxt) 그리고[fare_attributes.txt](#fare_attributestxt) )                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
agency_id not formatted as code
agency.txt not formatted as code
fare_attributes.txt not formatted as code